webapp/src/Service/SubmissionService.php

```diff
@@ -391,14 +356,14 @@ public function submitSolution(
         $contest,
         $language,
         array $files,
-        $source = null,
-        $juryMember = null,
+        ?string $source = null,
+        ?string $juryMember = null,
         $originalSubmission = null,
```

Comment:
add `?`?

Reply:
Nope becaue it is either null, int or a submission object, so we can't